src/graph/RaytraceNode.cpp

```diff
@@ -65,6 +65,10 @@ void RaytraceNode::enqueueExecImpl()
 		data->resize(raysNode->getRayCount(), false, false);
 	}
 
+	mrSamples.resize(MULTI_RETURN_BEAM_SAMPLES * raysNode->getRayCount());
+	mrFirst.resize(raysNode->getRayCount());
+	mrLast.resize(raysNode->getRayCount());
```

Comment:
No need to be allocated if we do not compute multi-return

Reply:
Good catch. For now, I cannot check it, but I added a TODO.